jdk/src/share/classes/java/util/Random.java

```diff
@@ -98,28 +98,13 @@
     static final String BadSize  = "size must be non-negative";
 
     /**
-     * Creates a new random number generator. This constructor sets
-     * the seed of the random number generator to a value very likely
-     * to be distinct from any other invocation of this constructor.
+     * Creates a new random number generator. Determinism
+     * requires that we always use the same seed.
      */
     public Random() {
-        this(seedUniquifier() ^ System.nanoTime());
+        this(8006694155008225895L);
```

Comment:
Where does this number come from?

Reply:
I generated it one day using the above algorithm, and then hardcoded it.